lib/fizzy/parser.cpp

```diff
@@ -249,7 +249,11 @@ inline parser_result<Code> parse(const uint8_t* pos, const uint8_t* end)
 
     const auto [locals_vec, pos2] = parse_vec<Locals>(pos1, end);
 
-    auto result = parse_expr(pos2, end);
+    auto [result, pos3] = parse_expr(pos2, end);
```

Comment:
Could name it `[code, pos3]` not confuse with the whole tuple's type `parser_result`

Reply:
Yeah, I've used that initially, but realised most other functions use `result`. I can change it.

Reply:
Changed.